site/guide/integrations/integrations-examples.qmd

```diff
@@ -3,20 +3,13 @@ title: "Integrations examples"
 date: last-modified
 ---
 
-Adapt these usage examples for your own workflows to interact with external systems.
+Adapt these usage examples for your own workflows or connections to interact with external systems.
 
 - [Create a Jira ticket](#create-a-jira-ticket)
 - [Create a ServiceNow incident](#create-a-servicenow-incident)
 - [Start workflows via webhook](#start-workflows-via-webhook)
 - [Add webhook step triggers](#add-webhook-step-triggers)
-
-These examples use HTTP request and webhook steps to integrate with external systems:
-
-- An *HTTP request* step sends data from {{< var vm.product >}} to another service. A workflow can notify a partner tool when a model reaches a certain state or request an update in an external system. This pattern is useful when {{< var vm.product >}} owns the event and needs to push it outward.
-
-- A *webhook* step does the opposite. Another system sends a POST request to  start a {{< var vm.product >}} workflow or to continue a workflow that is paused while waiting for a webhook to continue. This lets external tools trigger actions inside {{< var vm.product >}} when an event occurs on their side.
-
-Together these steps support event-driven data exchange that allows external systems to interact with the {{< var validmind.product >}}.
+- [Synchronize the model inventory with AWS SageMaker](#synchronize-the-model-inventory-with-aws-sagemaker)
```

Comment:
So we don't actually synchronize. Setting up the integration only allows you to make a link from model to model. So we should not say synchronize and we should show how you perform a linkage and what the result looks like.

Reply:
@nibalizer I'll ping you to chat, but this reference synchronizing is in our [frontend code](https://github.com/validmind/frontend/blob/main/src/pages/Settings/Integrations/Marketplace.tsx#L100), introduced by https://github.com/validmind/frontend/commit/fd2a08e9a975cf58b24d729e5e4b80b731830e47. 

<img width="904" height="743" alt="Capto_ 2025-11-25_09-28-19_am" src="https://github.com/user-attachments/assets/e5ec3492-31f0-427d-bf99-4de730b739d7" />